#Lambda Red Recombination
##Primer Design for Lambda Red Recombination

Target site for homologous recombination in MG1655: tonB-yciL intergenic region

Genome location: 1,308,593 → 1,309,832

<p dir="ltr">
    <span style="font-size:100%;font-family:Arial;color:#000000;background-color:#ea9999;">yciL</span>
    <span style="font-size:100%;font-family:'Arial';color:#000000;">, </span>
    <span style="font-size:100%;font-family:Arial;color:#000000;background-color:#9BCDFF;">tonB</span>
    <span style="font-size:100%;font-family:'Arial';color:#000000;">, </span>
    <span style="font-size:100%;font-family:Arial;color:#000000;background-color:#ffd966;">Homologous recombination arm</span>
    <span style="font-size:100%;font-family:'Arial';color:#000000;">, </span>
    <span style="font-size:100%;font-family:Arial;color:#000000;background-color:#93c47d;">Homologous recombination arm</span>
</p>
<p dir="ltr"><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#ea9999;">tcaGAACACTTTCTTAAATGGTTTCACTGAAACGTGTTCATAGACTCCTGCCGCTACGTACGGGTCAGCATCGGCCCAGGCCTGAGCTGCTTCCAGCGATTCAAATTCAGCAATAACGGTTGAGCCAGTAAATCCCGCAGCCCCTGGATCGTTACTGTCTACCGCTGGCATTGGACCAGCTGTCAACAAACGACCTTCATCATGCAGTAACTGTAAACGTGCTAAATGTGCCGGACGAACGGAAAGGCGTTTTTCGAGGCTATCAGCTTTGTCTTGAGCGTAAATAACATACAAcac</span><span style="font-size:100%;font-family:'Courier New';color:#000000;">GGGCACAACTCCTTGTTCGGGAAAGTTGTAAAGTACGTTATTTGAAAGGGCGAAGATCTGCAACGGAA</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#ffd966;">AGATGATGTCTTTGTTAAGGCCATGCATAAAGTAAGGGTAATTACGCCAA</span><span style="font-size:100%;font-family:'Courier New';color:#000000;">AAATGAC</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#93c47d;">ATTTTCACTGATCCTGATCGTCTTGCCTTATTGAATATGATTGCTATTTG</span><span style="font-size:100%;font-family:'Courier New';color:#000000;">CATTTAAAATCGAGACCTGGTTTTTCTACTGAAATGATTATGACTTCA</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#9BCDFF;">atgACCCTTGATTTACCTCGCCGCTTCCCCTGGCCGACGTTACTTTCGGTCTGCATTCATGGTGCTGTTGTGGCGGGTCTGCTCTATACCTCGGTACATCAGGTTATTGAACTACCTGCGCCTGCGCAGCCGATTTCTGTCACGATGGTTACGCCTGCTGATCTCGAACCGCCACAAGCCGTTCAGCCGCCACCGGAGCCGGTGGTAGAGCCAGAACCGGAACCTGAGCCGATCCCCGAACCGCCAAAAGAAGCACCGGTGGTCATTGAAAAGCCGAAGCCGAAACCTAAGCCAAAACCGAAGCCGGTGAAAAAGGTACAGGAGCAGCCAAAACGCGATGTCAAACCCGTAGAGTCGCGTCCGGCATCACCGTTTGAAAATACGGCACCGGCACGCCTGACATCAAGTACAGCAACGGCTGCAACCAGCAAGCCGGTTACCAGTGTGGCTTCAGGACCACGCGCATTAAGCCGTAATCAGCCGCAGTATCCGGCACGAGCACAGGCATTGCGCATTGAAGGGCAGGTTAAAGTTAAATTTGACGTTACGCCGGATGGTCGCGTGGATAACGTACAAATCCTCTCAGCCAAGCCTGCGAACATGTTTGAGCGTGAGGTGAAAAATGCGATGCGCAGATGGCGTTATGAGCCGGGTAAGCCAGGCAGTGGGATTGTGGTGAATATCCTGTTTAAAATTAACGGCACCACCGAAATTCAGtaa</span><span style="font-size:100%;font-family:'Courier New';color:#000000;">GCAGAAAGTCAAAAGCCTCCGACCGGAGGCTTTTGACTATTACTCAACAGGTAAGGCGCGAGGTTTTCCTTCAGGATCAACCGCGACATA</span></p>
<p>&nbsp;</p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">Example linear DNA cassette to introduce into the tonB-yciL intergenic region:</span></p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">5’</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#ffd966;">AGATGATGTCTTTGTTAAGGCCATGCATAAAGTAAGGGTAATTACGCCAA</span><span style="font-size:100%;font-family:Arial;color:#000000;">&lt;Promoter Seq&gt;&lt;RBS Seq&gt;&lt;ORF&gt;&lt;Terminator Seq&gt;</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#93c47d;">ATTTTCACTGATCCTGATCGTCTTGCCTTATTGAATATGATTGCTATTTG</span><span style="font-size:100%;font-family:Arial;color:#000000;"> 3’</span></p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">Primer sequence:</span></p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">1. 5’</span><span style="font-size:100%;font-family:'Courier New';color:#000000;background-color:#ffd966;">AGATGATGTCTTTGTTAAGGCCATGCATAAAGTAAGGGTAATTACGCCAA</span><span style="font-size:100%;font-family:Arial;color:#000000;">&lt;20-30bp of PCR template binding sequence&gt;</span></p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">2. (Reverse complement of homologous region highlighted in green)</span></p>
<p dir="ltr"><span style="font-size:100%;font-family:Arial;color:#000000;">5’</span><span style="font-size:100%;font-family:Arial;color:#000000;background-color:#93c47d;">CAAATAGCAATCATATTCAATAAGGCAAGACGATCAGGATCAGTGAAAAT</span><span style="font-size:100%;font-family:Arial;color:#000000;">&lt;20-30bp of PCR template binding sequence&gt;</span></p>

 
##Genome Engineering protocol

###Prepare linear DNA substrate for recombineering
* Measure the concentration of salt-free PCR product.

###Prepare fresh cells

1. Inoculate EcNR2 (or other strain) from a single colony into a falcon culture tube of 5 ml LB (LB lennox preferred for EcNR2) medium. Incubate the culture at 32 °C overnight with aeration. Maintain antibiotic selection.
 

###Induction of λ recombination genes

2. Add 0.5 ml of the overnight culture into each of 2 35 ml volumes of LB medium in 125-ml Erlenmeyer flasks. (Supplement with appropriate antibiotic to maintain plasmids, if needed). Dilute the overnight culture by at least 70-fold.
3. Place the flask in the 30 °C and grow cells with shaking for about 2 h or until the cells reach an OD550 of 0.55–0.65. The growth time may vary with different strains. It is important not to overgrow the cells, as stationary-phase cells are not optimal for recombineering.
4. Place one of the flasks to shake in a 42°C H2O bath; keep the other flask at 30 °C. Shake for 15 min at 200-220 r.p.m. The culture at 42 °C is induced for recombination functions and the 30 °C culture is negative control.

###Prepare electrocompetent cells

5. Immediately after the 15-min induction, rapidly chill both cultures in an ice-water slurry; swirl the flasks gently to cool down quickly. Sit on ice for 10 min. Label and chill two 50-ml centrifuge tubes for each set of induced cells and uninduced cells. Set the temperature of centrifuge at 0 °C simultaneously.
6. Transfer both of the induced cultures and uninduced cells to the 50-ml centrifuge tubes and centrifuge at 4,750 r.p.m. for 7 min at 0 °C. Using sterile technique, aspirate or pour off supernatant.
7. Add 1 ml of ice-cold sterile distilled H2O to the cell pellet and gently suspend cells with a large pipette tip (do not vortex).
8. Add another 20 ml of ice-cold distilled H2O to each tube,  seal and gently invert to mix, again without vortexing. Centrifuge tubes again as mentioned in Step 6 .
9. Promptly decant the 20-ml supernatant very carefully from the soft pellet in each tube and gently suspend each cell pellet in 1 ml of ice-cold distilled H2O. Remove tubes from the centrifuge promptly at the end of spin.
* The pellet is very soft and care should be taken not to dislodge it or lose the cells, especially when processing multiple tubes. If necessary, leave a little supernatant in the tube.
10. Repeat the same washing step with ice-cold distilled H2O again and aspirate supernatant, being extremely careful with the pellet. Prepare pre-chilled microcentrifuge tubes.
11. Suspend the cell pellet in 200 μl of sterile cold distilled H2O and aliquot to 8 microcentrifuge tubes with 25 μl in each.  Keep these microcentrifuge tubes on ice until used.
* For highest efficiency, use freshly processed cells.

###Perform electroporation of linear targeting substrate DNA

12. Chill the desired number of labeled electroporation cuvettes on ice.
13. In labeled microcentrifuge tubes on ice, add target DNA ~50 ng-100 ng of salt-free PCR fragment. Mix the culture thoroughly.
14. Transfer the DNA–cell mixes to chilled electroporation cuvettes as quickly as possible and introduce the DNA into the cells by electroporation (1800 V). Resulting time constant should be 4-6.
15. Immediately after electroporation, add 1 ml of SOC medium to the cuvette. After the electroporations are completed, transfer the electroporation mixes to sterile culture tubes and incubate with shaking at 30 °C for 2~3 h.

###Outgrowth and selection of recombinant clones by antibiotic resistance selection

16. Plate 100–200 μl of the undiluted cells on selective plates and incubate at 30°C overnight. For the control cultures, plate 200 μl of the undiluted culture on a single selective plate.
17. Incubate plates until colonies appear (generally 22–24 h) at 30 °C.
* Do not add antibiotic for host maintenance while outgrowing or selecting the recombinant on plates.
* For confirming insertion of a dsDNA, such as a drug cassette, use two pairs of primers, each pair having one primer outside the targeted flanking region and one primer in the inserted DNA, and amplify the two junctions. Another PCR, using the two outside flanking primers, should also be performed to confirm the absence of the gene to be removed, thus ruling out the possibility of a duplication event.
 

##Notes:

When using tetracycline for selection, add only 12.5 ug/mL to the plate because only one copy of the tetAR operon is left in each cell after chromosomal integration. Prepare tetracycline plate on the day for transformation. Negative controls grow on tetracycline plate after 48 h of incubation (it is possible that tetracycline degraded at 30°C).

##Sources:

 
```
Sharan, S. K., L. C. Thomason, S. G. Kuznetsov and D. L. Court. (2009). Recombineering: a homologous recombination-based method of genetic engineering. NATURE PROTOCOLS, 4(2): 206-223.

Liu, P., N. A. Jenkins and N. G. Copeland. 2003. A Highly efficient recombineering-based method for generating conditional knockout mutations. Genome Research, 13: 476-484.

Mosberg, J. A., M. J. Lajoie and G. M. Church. 2010. Lambda red recombineering in Escherichia coli occurs through a fully single-stranded intermediate. Genetics, 186: 791-799

Baba, T., et al. 2006. Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular System Biology, 2:2006.0008.

Datsenko, K. A. and B. L. Wanner. 2000. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. PNAS, 97(12): 6640-6645.   

Nyerges, A. et al. 2016. A highly precise and portable genome engineering method allows comparison of mutational effects across bacterial species. PNAS, 113(9): 2502-2507 ```